ansible/gcp/start_migration.yaml

```diff
@@ -14,7 +14,7 @@
     shell: "python3 /tmp/start.py {{ bucket }} {{ access_key }} {{ secret_key }} '{{ mongodb }}' {{ project }}"
     register: st
     become_user: root
-    become: True
+    become: true
```

Comment:
This become is redundant because become is set to true for the whole play on the top